### PR TITLE
[geometry] Add y2023 first cut and y2022a production geometries

### PR DESCRIPTION
**Pull request by klendathu2k**

### CODE REVIEW

File: StRoot/StChain/GeometryDbAliases.h [hunk 1]
```diff
@@ -157,6 +157,9 @@ static const DbAlias_t fDbAlias[] = {// geometry  Comment            old
   {"y2021a",      20201215,     1, "y2021a",   "y2021 first production geometry, AgML,xgeometry"},   
   
   {"y2022",       20211015,     0, "y2022",    "y2022 first cut geometry, AgML,xgeometry"},   
+  {"y2022a",      20211015,     1, "y2022a",   "y2022 production geometry, AgML,xgeometry"},  
+
+  {"y2023",       20221215,     0, "y2032",    "y2023 first cut geometry, AgML,xgeometry"},
```
Comment (plexoos) [inline StRoot/StChain/GeometryDbAliases.h]: Is the forth argument correct?

**Review by plexoos — APPROVED** [review]

File: StRoot/StChain/GeometryDbAliases.h (hunk 1 above)
Comment (klendathu2k) [reply StRoot/StChain/GeometryDbAliases.h]: It is now ;)  Thanks!

**Comment by plexoos** [comment]
Has been pending for 7 days. Shall we assume there will be no further comments from the co-maintainers?

**Comment by klendathu2k** [comment]
Yes.  Let's got ahead and merge.

**Comment by plexoos** [comment]
Ok, but you just triggered CI jobs by merging the main branch... Let's wait for them to finish.

**Comment by klendathu2k** [comment]
Agreed.

**Pull request merged**